presentation/showoff/Introduction/01_slide.md

```diff
@@ -0,0 +1,18 @@
+!SLIDE[bg=_images/backgrounds/white_bg.png]
```

Comment:
how late in the process can this be edited?

Reply:
Unsure what you're getting at here. Can you clarify?

Reply:
Whoever's presenting should have their info at the front. Not sure if we're intending to publish on the web or have people live presenting it.

An attribution for us all at some point inside would be nice as well.